### DISCUSSION TITLE
[Request / Idea] Line number bookmarks

### DISCUSSION THREAD

**Discussion by internet-police** [tsujan/FeatherPad#783] [Ideas]
Suppose i have to jump a lot between the lines 500 2000 and 5352.
So instead of scrolling like a mad man or using the search function,
it would be very nice if i could just click on a bunch of line numbers or press something like ctrl+space or whatever, to mark the line.
And then press something else like ctrl+tab to cycle through those marked lines.

If it would be just stored per session we dont need to create extra files.

Stay safe.

**Comment by tsujan** [comment]
> Suppose i have to jump a lot between the lines...

Well, I do it on a regular basis, by opening the same file in two, three,… tabs. That's efficient, fast and, most importantly, safe.
